browser/net/BUILD.gn

```diff
@@ -39,6 +39,12 @@ source_set("net") {
     "//net",
   ]
 
+  if (!is_android) {
```

Comment:
Can android app not use strings from `//brave/app:brave_generated_resources_grit` @simonhong @bridiver ? If not, and `//brave/browser/net` will be used by android build, how do we get those string resources declared for android - do they need to be added separately?

Reply:
@petemill I might be wrong - I referred `packed_resources` target that is excluded from android.
After searching `generated_resources` in chrome and brave, I think `//brave/app:brave_generated_resources_grit` could be in `is_android` also. @bridiver

Reply:
a lot of these guards were just to get the build working. I didn't have time to dig into each one so @SergeyZhukovsky is working on fixing actual runtime issues related to these

Reply:
most likely we need to split these grd file up for things like extensions

Reply:
@petemill what was the build error on android?